db/migrate/20160802145938_set_generic_subtype_on_service_template.rb

```diff
@@ -0,0 +1,15 @@
+class SetGenericSubtypeOnServiceTemplate < ActiveRecord::Migration[5.0]
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time('Set generic_subtype to custom on ServiceTemplate') do
+      ServiceTemplate.update_all(:generic_subtype => "custom")
```

Comment:
@lfu We should only be updating generic service_templates.  You need to include `.where(:prov_type => "generic")`

Same for the down migration.